transbank/transaccion_completa/schema.py

```diff
@@ -0,0 +1,82 @@
+from marshmallow import Schema, fields
+from transbank.common.schema import CardDetailSchema
+
+
+class CreateTransactionRequestSchema(Schema):
+    buy_order = fields.Str()
+    session_id = fields.Str()
+    amount = fields.Float()
+    card_number = fields.Str()
+    cvv = fields.Str()
+    card_expiration_date = fields.Str()
+
+
+class CreateTransactionResponseSchema(Schema):
+    error_message = fields.Str()
+    token = fields.Str()
+
+
+class CommitTransactionRequestSchema(Schema):
+    id_query_installments = fields.Str()
+    deferred_periods_index = fields.Str()
+    grace_period = fields.Str()
+
+
+class CommitTransactionResponseSchema(Schema):
+    error_message = fields.Str()
+    # vci = fields.Raw()
+    amount = fields.Float()
+    status = fields.Str()
+    buy_order = fields.Str()
+    session_id = fields.Str()
+    card_detail = fields.Nested(CardDetailSchema, many=False)
+    accounting_date = fields.Str()
+    transaction_date = fields.Str()
+    authorization_code = fields.Str()
+    payment_type_code = fields.Str()
+    response_code = fields.Float()
+    installments_number = fields.Float()
+
+
+# installments_amount = fields.Float()
```

Comment:
same here?

Reply:
They aren't, they're mostly me trying to make sense between what's stated on TBKs docs and what actually is returned from their APIs (since by default python throws if the key is not found)